### ISSUE TITLE
hi,are you sure your focal loss equals to -alpha*(1-pt)^gamma*log(pt)

### ISSUE THREAD

**Issue by li10141110** [ailias/Focal-Loss-implement-on-Tensorflow#1]
same as above and thank you for your attention in advance

**Comment by ChiefGodMan** [comment]
Yes, it is. And we need to take care of two conditions(true or false) just like cross_entropy loss function.
